File: TYPESCRIPT/Cap03/c03ex01.ts
// c03ex01.ts

import * as rls from 'readline-sync';

let A: number;
let B: number;
let X: number;

console.log('ADICAO DE NUMEROS');
console.log();

A = rls.questionInt('Entre o 1o. valor numerico inteiro: ');
B = rls.questionInt('Entre o 2o. valor numerico inteiro: ');

X = A + B;

console.log('Resultado da adicao = ', X);
console.log();

process.stdout.write('Tecle <Enter> para encerrar... ');
rls.question();

File: LUA/Cap03/c03ex01.lua
-- c03ex01.lua

io.write("Entre o 1o. valor numerico inteiro: ")
a = tonumber(io.read())

io.write("Entre o 2o. valor numerico inteiro: ")
b = tonumber(io.read())

x = a + b

print("Resultado = " .. x)

print()
io.write("Tecle <Enter> para encerrar... ")
io.read()
